3rd_party_deps.yml

```diff
@@ -1,7 +1,26 @@
 :environment:
   - :HE_WOLFSSL_SOURCE: https://github.com/wolfSSL/wolfssl
-  - :HE_WOLFSSL_TAG: v5.6.4-stable
-  - :HE_WOLFSSL_CONF_FLAGS: --enable-tls13 --disable-oldtls --enable-static --enable-singlethreaded --enable-dtls --enable-dtls13 --enable-secure-renegotiation --enable-sp=yes,4096 --disable-shared --enable-dtls-mtu --disable-sha3 --disable-dh --enable-curve25519 --disable-examples --disable-sys-ca-certs --enable-sni --disable-benchmark --enable-dtls-frag-ch
+  - :HE_WOLFSSL_TAG: v5.6.6-stable
+  - :HE_WOLFSSL_CONF_FLAGS: >-
```

Comment:
I noticed this was split into multiple lines for readability, which is good. But it was also re-ordered a bit, making it hard to visually diff.
 
 If we're going to re-order this anyway, can I suggest sorting it alphabetically so that there's a deterministic way to list these flags and make diffs easier to read in the future?

Reply:
Good point. Fixed.